views/account/account.handlebars

```diff
@@ -145,7 +145,7 @@
 				<div class="setting-card">
 					<h2 class="header">{{ locale.account.settings.settingCards.discord }}</h2>
 						{{#if discordUser}}
-					<p>{{ locale.account.settings.settingCards.connectedToDiscord }} {{ discordUser.username }}#{{ discordUser.discriminator }}</p>
+					<p>{{ locale.account.settings.settingCards.connectedToDiscord }} {{ discordUser.username }}{{#ifCond discordUser.discriminator '#0'}}{{else}}#{{ discordUser.discriminator }}{{/ifCond}}</p>
```

Comment:
- There is no handlebars function called `ifCond`. This crashes
- Even if this didn't crash, `discordUser.discriminator` is a number, it does not contain the `#`, which is why it is added here. These checks would always fail since you seem to be checking for `#0` and not `0`
- We prefer to use full blocks for if statements instead of single lines. See the `#if` checks earlier in this same view for checking the users `environment` status

Reply:
should be fixed, however i was unable to rehost the website as i was missing critical files for the account server and didnt know how to get them